test/e2e/kind.sh

```diff
@@ -19,20 +19,17 @@ set -euo pipefail
 
 source test/e2e/aliases.sh
 
-# images built for kind v0.20.0
-# https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0
+# images built for kind v0.24.0
```

Comment:
Could you upgrade the versions of kind and kubectl installed in test/e2e/install_dependencies.sh?